airflow/providers/apache/beam/triggers/beam.py

```diff
@@ -166,7 +168,7 @@ def __init__(
         project_id: str | None = None,
         location: str | None = None,
         job_name: str | None = None,
-        gcp_conn_id: str | None = None,
+        gcp_conn_id: str = "google_cloud_default",
```

Comment:
Is it okay to set this as google connection since this is apache beam trigger? :)

Reply:
A good point. I am not sure, but I think it's fine. Because I specify here a default value for `gcp_conn_id` and, also, we use the same approach in `BeamOperators`: https://github.com/apache/airflow/blob/main/airflow/providers/apache/beam/operators/beam.py#L519